sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -3136,12 +3141,12 @@ checkDevSupportDeviceRequirements(const device &Dev,
 } // namespace _V1
 } // namespace sycl
 
-extern "C" void __sycl_register_lib(pi_device_binaries desc) {
+extern "C" void __sycl_register_lib(sycl_device_binaries desc) {
```

Comment:
I'm not necessarily asking for this PR, but can we document `__sycl_register_lib` somewhere?  It is one of the lynchpins between the FE and SYCL and has exactly zero comments or docs anywhere in the code base.

Reply:
Yes, I agree that we a huge gap in our documentation, but I think that it should be a separate PR